src/components/Volunteer/EventVolunteer.jsx

```diff
@@ -2,15 +2,17 @@ import React from "react";
 import Select from "react-select";
 import { navigate } from "gatsby";
 import reducer from "../../reducers/commonReducer";
-import { createEventVolunteer } from "../../services/volunteerService";
+import { addEventVolunteer } from "../../services/volunteerService";
 import { Input, Button } from "../../commons/Form";
 import { getAll } from "../../services/collegeServices";
 import { toast } from "../../actions/toastActions";
 
 class EventVolunteer extends React.Component {
     ADD_VOLUNTEER = "Add Volunteer";
+    ADDING_VOLUNTEER = "Adding...";
     state = {
-        buttonText: this.ADD_VOLUNTEER
+        buttonText: this.ADD_VOLUNTEER,
+        count: 6
```

Comment:
A comment on why this starts with a 6

Reply:
Old (unnecessary) code , when default number of volunteers was 6. Thanks for point this out. 👍